descriptors/ModuleDescriptor-template.json

```diff
@@ -92,6 +92,14 @@
           "methods": ["GET"],
           "pathPattern": "/invoice/invoice-number",
           "permissionsRequired": ["invoice.invoice-number.item.get"]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/voucher/vouchers/{id}",
```

Comment:
Maybe it is worth placing everything related to vouchers in a separate interface.

Reply:
Done